Documentation/UsingSetting/_Sets/_TcaOverridesSystemplateV12.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+use TYPO3\CMS\Core\Information\Typo3Version;
+use TYPO3\CMS\Core\Utility\ExtensionManagementUtility;
+use TYPO3\CMS\Core\Utility\GeneralUtility;
+
+defined('TYPO3') or die();
+
+call_user_func(function()
```

Comment:
Why call_user_func and not a self-executing anonymous function?

    (function() {
        // ...
    })();